### DISCUSSION TITLE
Help with the configuration

### DISCUSSION THREAD

**Discussion by tripdog** [flyntwp/flynt#410] [Q&A] [answered]
Hey all!

I don't think my installation is quite right. I've gone through all of the steps and activated the theme. But when I run npm run build I get these errors: 

Oops! Something went wrong! :(
ESLint: 7.32.0
ESLint couldn't find the plugin "eslint-plugin-import".
(The package "eslint-plugin-import" was not found when loaded as a Node module from the directory "/Users/tomphillips/Dropbox/Code Repo/bootstrap-5.0.2".)

Then when I run NPM start I get these errors:
node: 15.14.0
(node:74023) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
(Use `node --trace-deprecation ...` to show where the warning was created)
[Browsersync] Proxying: http://palbarling.local
[Browsersync] Access URLs:
 ------------------------------------
  Local: https://localhost:3000
 External: https://192.168.50.75:3000
 ------------------------------------
[Browsersync] Watching files...
 ERROR  Failed to compile with 1 errors                                             9:17:59 PM

 error  in ./assets/main.scss

Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: An @import loop has been found

Is anyone able to offer some advice?
I can see the theme and every thing is unstlyed which at first, I thought made sense until I read further in the quick start that said there are some base scss styles to play with. Initially I had set up my WordPress installation with Local By FlyWheel ...

**Comment by steffenbew** [comment, marked as answer]
@tripdog Hey Tom, it looks like you're running node v15.14.0. Could you please try node v12 instead? Please delete the node_modules before reinstalling.

**Reply by steffenbew** [reply]
Please also see #397.

**Comment by tripdog** [comment]
Hi Steffen-

Thanks for the advice. Some errors have indeed been fixed but there is still something not quite right with ES Lint. I have no idea why it's trying to reference other projects in my projects directory.  Here are the errors:

node: 12.22.7
Oops! Something went wrong! :(
ESLint: 7.32.0
ESLint couldn't find the config "react-app" to extend from. Please check that the name of the config is correct.

**The config "react-app" was referenced from the config file in "/Users/tomphillips/Dropbox/Courses/JS_Bootcamp/Week 11/react-hooks/2-set-and-update-state-with-useState/state-hook/package.json".**

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! flynt@1.4.0 build: `run-s --silent checkVersion clean lint build:production build:rev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the flynt@1.4.0 build script.

Thanks for your help. :-)

**Reply by steffenbew** [reply]
Hi Tom, that's interesting and unexpected to me. Can you confirm that you're running the install command directly from within the flynt theme directory?
What I would naturally do at this point, is deleting the whole project from my hard drive, and downloading and reinstalling it from scratch again.

**Reply by tripdog** [reply]
Ok great, will do.
